Tasks/AzureRmWebAppDeployment/operations/KuduServiceUtility.ts

```diff
@@ -96,7 +99,7 @@ export class KuduServiceUtility {
         return deploymentID;
     }
 
-    public async deployWebPackage(packagePath: string, physicalPath: string, virtualPath: string, appOffline: boolean): Promise<void> {
+    public async deployWebPackage(packagePath: string, physicalPath: string, virtualPath: string, appOffline?: boolean): Promise<void> {
```

Comment:
Is this function used anywhere? If not, should we not remove it?

Reply:
we use it for Windows App - Linux agent